admin_app_2/src/app/content/page.tsx

```diff
@@ -0,0 +1,206 @@
+import { Box, Button, CircularProgress, Grid, Typography } from "@mui/material";
+import Alert from "@mui/material/Alert";
+import Snackbar from "@mui/material/Snackbar";
+import Link from "next/link";
+import { useSearchParams } from "next/navigation";
+import React from "react";
+
+export default ContentScreen;
+
+function ContentScreen() {
+  return (
+    <Layout.FlexBox alignItems="center" flexDirection={"column"}>
+      <Layout.Spacer multiplier={3} />
+      <CardsView />
+    </Layout.FlexBox>
+  );
+}
+
+const CardsView = () => {
+  const [displayLanguage, setDisplayLanguage] = React.useState<string>(
+    LANGUAGE_OPTIONS[0].label,
+  );
+  return (
+    <Layout.FlexBox width={"100%"}>
+      <Layout.Spacer multiplier={1} />
+      <CardsUtilityStrip />
+      <Layout.Spacer multiplier={1} />
+      <CardsGrid displayLanguage={displayLanguage} />
+    </Layout.FlexBox>
+  );
+};
+
+const CardsUtilityStrip = () => {
+  return (
+    <Layout.FlexBox
+      key={"utility-strip"}
+      flexDirection={"row"}
+      justifyContent={"flex-right"}
+      alignItems={"right"}
+      sx={{
+        display: "flex",
+        alignSelf: "flex-end",
+        px: sizes.baseGap,
+      }}
+      gap={sizes.baseGap}
+    >
+      <Link href="/content/edit">
+        <Button variant="contained">
+          <Add />
+          New
+        </Button>
+      </Link>
+    </Layout.FlexBox>
+  );
+};
+const CardsGrid = ({ displayLanguage }: { displayLanguage: string }) => {
+  const [page, setPage] = React.useState<number>(1);
+  const [max_pages, setMaxPages] = React.useState<number>(1);
+  const [cards, setCards] = React.useState<any[]>([]);
+  const [isLoading, setIsLoading] = React.useState<boolean>(true);
+
+  const searchParams = useSearchParams();
+  const action = searchParams.get("action") || null;
+  const content_id = Number(searchParams.get("content_id")) || null;
+
+  const getSnackMessage = (
+    action: string | null,
+    content_id: number | null,
+  ): string | null => {
+    if (action === "edit") {
+      return `Content id ${content_id} updated`;
+    } else if (action === "add") {
+      return `Content id ${content_id} created`;
+    }
+    return null;
+  };
+
+  const [snackMessage, setSnackMessage] = React.useState<string | null>(
+    getSnackMessage(action, content_id),
+  );
+
+  const MAX_CARDS_PER_PAGE = 4;
+
+  const [refreshKey, setRefreshKey] = React.useState(0);
+  const triggerRefresh = () => {
+    setRefreshKey((oldKey) => oldKey + 1);
+  };
+
+  React.useEffect(() => {
+    apiCalls.getContentList().then((data) => {
+      setCards(data);
+      setMaxPages(Math.ceil(data.length / MAX_CARDS_PER_PAGE));
+      setIsLoading(false);
+    });
+  }, [refreshKey]);
+
+  if (isLoading) {
+    return (
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "row",
+          justifyContent: "center",
+          alignItems: "center",
+          height: "100vh",
```

Comment:
This places too low on the page.

```suggestion
          height: "100vh",
```